pkg/scheduler/framework/plugins/dynamicresources/dynamicresources.go

```diff
@@ -1016,30 +1016,24 @@ func (pl *dynamicResources) lookupClassParameters(logger klog.Logger, class *res
 }
 
 func (pl *dynamicResources) lookupClaimParameters(logger klog.Logger, class *resourcev1alpha2.ResourceClass, claim *resourcev1alpha2.ResourceClaim) (*resourcev1alpha2.ResourceClaimParameters, *framework.Status) {
-	defaultClaimParameters := resourcev1alpha2.ResourceClaimParameters{
-		Shareable: true,
-		DriverRequests: []resourcev1alpha2.DriverRequests{
-			{
-				DriverName: class.DriverName,
-				Requests: []resourcev1alpha2.ResourceRequest{
-					{
-						ResourceRequestModel: resourcev1alpha2.ResourceRequestModel{
-							// TODO: This only works because NamedResources is
-							// the only model currently implemented. We need to
-							// match the default to how the resources of this
-							// class are being advertized in a ResourceSlice.
-							NamedResources: &resourcev1alpha2.NamedResourcesRequest{
-								Selector: "true",
-							},
-						},
-					},
-				},
-			},
-		},
-	}
+	if claim.Spec.ParametersRef == nil && class.DefaultClaimParametersRef == nil {
```

Comment:
@pohly - Are these 2 the only cases that we should be concerned about? Let me know if I'm missing something.